go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/virtblocks/virtblocks
+
+go 1.12
```

Comment:
Adding module support should be its own commit.

Reply:
Done.